algebra/u256/src/u256.rs

```diff
@@ -83,27 +85,6 @@ impl Ord for U256 {
     }
 }
 
-#[cfg(any(test, feature = "quickcheck"))]
```

Comment:
Don't we need these around untill everything is ported over?

Reply:
this PR migrates the U256 crate to proptest, so we don't need it anymore in this crate anymore. The prime-field crate effectively re-implements U256::arbitrary.